proofs/cbmc/mldsa_shake128_stream_init/mldsa_shake128_stream_init_harness.c

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2025 The mldsa-native project authors
+// SPDX-License-Identifier: Apache-2.0
+
+#include "params.h"
+#include "symmetric.h"
+
+void harness(void)
+{
+  keccak_state *s;
+  const uint8_t seed[MLDSA_SEEDBYTES];
```

Comment:
Blocker: This must be `const uint8_t *seed`. Otherwise, the harness always calls `mldsa_shake128_stream_init` with a valid `seed` pointer. Instead, it should call the function-under-proof with a priori unconstrained arguments, which are then _only_ constrained by the function preconditions.

To check, try removing the validity precondition for `seed` -- I suspect the proof will still go through.

We should double-check whether this is done elsewhere, too.

Reply:
ok fixed!